src/renderer/components/OperationsList/index.js

```diff
@@ -70,12 +72,17 @@ export class OperationsList extends PureComponent<Props, State> {
 
   state = initialState;
 
-  handleClickOperation = (operation: Operation, account: AccountLike, parentAccount?: Account) =>
+  handleClickOperation = (operation: Operation, account: AccountLike, parentAccount?: Account) => {
+    const currency = getAccountCurrency(account);
+    if (this.props.operationClickEvent) {
+      track(this.props.operationClickEvent, { currencyName: currency.name });
+    }
```

Comment:
mmh, why is this not covered by an event attached on MODAL_OPERATION_DETAILS modal? it should be the approach because there are other cases for this modal.